### ISSUE TITLE
Move AIMerger to its own repo

### ISSUE THREAD

**Issue by pavi2410** [mit-cml/appinventor-sources#2167]
The aimerger and aimerger_for_appinv_classic modules are separate applications and should be moved, as they are totally independent of the developments in this repo.

I also have a working version of AIMerger which uses Gradle. The advantage of this is that they are simple and easy to build and run.
https://github.com/pavi2410/appinventor-sources/tree/gradle/aimerger

**Comment by ewpatton** [comment]
> they are totally independent of the developments in this repo.

Yes and no. AIMerger is highly dependent on the project format. If we were to change the project format here, support would need to be introduced into AIMerger as well. As it stands, the version you check out with App Inventor will work for that version (and in this hypothetical example it would hold true). This is the same reason for moving the documentation into this repo rather than hosting it on our website. Otherwise, you end up with link rot and potentially lose access to the correct information for a particular version.

I don't think I've ever been inconvenienced for the fact that those two modules live in this repo. What prompted the desire to expunge them?

**Comment by pavi2410** [comment]
It was because while working on migrating build system to Gradle, I noticed that these modules were independent of the build chain. Also, these modules haven't had any significant change in so many years. So, I thought it'd be better to have them separated, and opened this issue to know your opinion.

But, I still doubt that aimerger_for_appinv_classic is required because it doesn't work for AI2.
